lib/oxidized/model/aoscx.rb

```diff
@@ -59,6 +59,7 @@ class Aoscx < Oxidized::Model
     cfg.gsub! /^(\d+\/\S+\s+\S+\s+)\d+\.\d+\s+C(.*)/, '\\1<hidden>\\2'
     cfg.gsub! /^(LC.*\s+)\d+\.\d+\s+(C.*)$/, '\\1 <hidden> \\2'
     cfg.gsub! /^(\S+\s+\S+\s+\s+\S+\s+)(slow|normal|medium|fast|max)(\s+\S+\s+\S+\s+)\d+/, '\\1<speed>\\3<rpm>'
+    cfg.gsub! /^(\d+\s+.+\s+)(\s\s\d+\.\d+)(\s+)(\s\s\d+\.\d+)$/, '\\1<power>\\3<power>'
```

Comment:
I managed to fix the outstanding issues mentioned in my comment by replacing this line with:

```ruby
    cfg.gsub! /^(\d+\s+.+\s+)(\s\s\d+\.\d+)(\s+)(\s\s\d+\.\d+)$/, '\\1<power>      <power>'
    cfg.gsub! /^Total Power Consumption\s+\d+\.\d+$/, 'Total Power Consumption                  <power>'
```

Reply:
Hello @alextremblay, thank you very much for your comment and checks. I will enhance regex to match the power consumption for up to a five digit number. My intent with the spaces that I am matching with the regex is, to keep the table formatting and not move / re-align the table row depending on the number of digits captured. I will document this also in the issue.

Reply:
I have now updated my regex to adequately replace up to four digit numbers (I guess no single switch can ever consume more than 10k Watts).

Reply:
@alextremblay I cannot see any output field "Total Power Consumption" as you stated in the output. Can you provide an example output? Does this only show up with switch stacks as a bottom line?

Reply:
Hi @alextremblay, I have now checked the documentation and added support for the 6400, 8400 switch series and chassis switches. I could not find total power consumption anywhere in the output, except with `detail` at the show output. However, I have added this since this may also happen in the future.